azure-defender-easm/label-known-easm-assets/service.py

```diff
@@ -11,6 +11,11 @@
 
 load_dotenv()
 
+logging.basicConfig(
+    level=logging.info, format="[%(asctime)s :: %(name)s :: %(levelname)s] %(message)s"
+)
```

Comment:
Level should use the `logging.INFO` level. Should call `basicConfig` before the client imports so they can use it too.